types/netvox/r711/meta.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "R711-Wireless Temperature and Humidity Sensor",
+  "name": "R711",
```

Comment:
changes in name will need to be done manually, therefore coordination when merging this is needed